### PR TITLE
add Ascend get_started

### PR DESCRIPTION
**Pull request by jinminxi104**
## Motivation

this pr added get_started doc for huawei ascend device

## Modification

added new get_started into doc folder

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/get_started/ascend/get_started.md [hunk 1]
```diff
@@ -1,15 +1,116 @@
...
+Set `device_type="ascend"`  in the `PytorchEngineConfig`:
+
+```python
+from lmdeploy import pipeline
+from lmdeploy import PytorchEngineConfig
+if __name__ == "__main__":
+    pipe = pipeline(“internlm/internlm2_5-7b-chat",
+                            backend_config = PytorchEngineConfig(tp=1, device_type="ascend"))
+    question = ["Shanghai is", "Please introduce China", "How are you?"]
+    response = pipe(question)    print(response)
```
Comment (lvhan028) [inline docs/en/get_started/ascend/get_started.md:67]: Let's put `print(response)` in another line

File: docs/en/get_started/ascend/get_started.md [hunk 2]
```diff
@@ -1,15 +1,116 @@
...
+```
+
 ### VLM inference
 
+Set `device_type="ascend"`  in the `PytorchEngineConfig`:
```
Comment (lvhan028) [inline docs/en/get_started/ascend/get_started.md:72]: redundant white spaces before `in`

File: docs/en/get_started/ascend/get_started.md [hunk 3]
```diff
@@ -1,15 +1,116 @@
...
+
+```python
+from lmdeploy import pipeline, PytorchEngineConfig
+from lmdeploy.vl import load_image
+if __name__ == "__main__":
+        pipe = pipeline(‘/path_to_InternVL2-2B',
```
Comment (lvhan028) [inline docs/en/get_started/ascend/get_started.md:78]: Let's use "OpenGVLab/InternVL2-2B" instead of "/path_to_InternVL2-2B"

Comment (lvhan028) [reply docs/en/get_started/ascend/get_started.md:78]: The indent should be 4 not 8

File: docs/en/get_started/ascend/get_started.md (hunk 1 above)
Comment (jinminxi104) [reply docs/en/get_started/ascend/get_started.md]: fixed

File: docs/en/get_started/ascend/get_started.md (hunk 2 above)
Comment (jinminxi104) [reply docs/en/get_started/ascend/get_started.md]: fixed

File: docs/en/get_started/ascend/get_started.md (hunk 3 above)
Comment (jinminxi104) [reply docs/en/get_started/ascend/get_started.md]: fixed

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**